src/Collection/Introductions.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Friendica\Collection;
+
+use Friendica\Collection;
+use Friendica\Model\Introduction;
+
+/**
+ * @property Introduction[] $models
+ */
+class Introductions extends Collection
```

Comment:
As said above, why don't we use the plain `Introduction[] $models` array directly instead of creating a second class for it? I think this could make it harder for others to understand the concept.

Reply:
See https://github.com/friendica/friendica/pull/8047#discussion_r362635411